### DISCUSSION TITLE
C-FIND and C-MOVE

### DISCUSSION THREAD

**Discussion by Blackglade** [nextgenhealthcare/connect#6105] [Frequently Asked Questions]
Hi,

In Mirth Connect would I be able to ingest HL7 messages and issue out DIMSE C-FIND and C-MOVE commands to other destinations?

**Comment by kirbykn2** [comment]
I would think GPT
yes, Mirth Connect can be configured for HL7 message ingestion and can potentially issue out DIMSE C-FIND and C-MOVE commands to other destinations, especially when integrated with comprehensive healthcare interoperability platforms or engines that support a wide range of data formats and communication protocols, including HL7, FHIR, DICOM, and others. For specific DICOM operations like C-FIND and C-MOVE, you might need to leverage external libraries or tools that specialize in DICOM communications, as Mirth Connect primarily focuses on HL7 and healthcare data interchange. Integrating Mirth Connect with such tools can facilitate the desired DICOM operations within your workflow

**Reply by Blackglade** [reply]
Did you just ChatGPT an answer? I was looking for something more specific?

**Reply by kirbykn2** [reply]
I know radiology orgs that use MC exclusively.  If your looking for specific help doing the work, I suggest finding a good consultant.
